lib/Doctrine/DBAL/Cache/ArrayStatement.php

```diff
@@ -20,7 +20,7 @@
 namespace Doctrine\DBAL\Cache;
 
 use Doctrine\DBAL\Driver\ResultStatement;
-use PDO;
+use Doctrine\DBAL\PDOCompat;
```

Comment:
Why not simply providing a `PDO` class via something like following? Much simpler, much better compatibility:

```json
{
    "autoload": {
        "classmap": [
            "path/to/PDO.php"
        ]
    }
}
```